internal/command/machine/update.go

```diff
@@ -92,6 +95,15 @@ func runUpdate(ctx context.Context) (err error) {
 		waitForAction = "stop"
 	}
 
+	out := io.Out
+	fmt.Fprintln(out, colorize.Yellow(fmt.Sprintf("Machine %s has been updated\n", machine.ID)))
+	fmt.Fprintf(out, "Instance ID has been updated:\n")
+	fmt.Fprintf(out, "%s -> %s\n\n", prevInstanceID, machine.InstanceID)
+	fmt.Fprintf(out, "Image: %s\n", machine.Config.Image)
+	fmt.Fprintf(out, "State: %s\n\n", machine.State)
```

Comment:
Oh! I think we need to reload the machine state here?

Reply:
oh, you're ttly right, we do!

Reply:
I assume that since we call wait for start/stop and we know what state it's in, we can just assume that state without having to make another call to flaps?

Reply:
Yea, I honestly think that's fine to do.